docs/components/camera/_index.md

```diff
@@ -173,6 +173,55 @@ For more information, see the [Go SDK Docs](https://pkg.go.dev/go.viam.com/rdk/c
 {{% /tab %}}
 {{< /tabs >}}
 
+### GetImages
+
+Get simultaneous images from different sensors, along with associated metadata.
+This should not be used for getting a time series of images from the same sensor.
```

Comment:
Is the docs more for users, or developers, or both? Because this statement of about how the method should not be used for getting a time series of images is much more a comment for a developer implementing GetImages. For a user, I think the phrase would be like "the multiple images returned from GetImages will never represent a time series of images"

Reply:
That sounds good. I was leaning on the python and go docs